sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TraceHelperTests.cs

```diff
@@ -274,19 +281,22 @@ public void ActivityWithEventCreatesTraceTelemetry()
             Assert.Equal(2, telemetryItems.Count());
             Assert.Equal("Message", telemetryItems[0].Name);
             Assert.Equal("Request", telemetryItems[1].Name);
-            Assert.Equal(eventName, (telemetryItems[0].Data.BaseData as MessageData).Message);
-            Assert.True((telemetryItems[0].Data.BaseData as MessageData).Properties.TryGetValue("key1", out var value));
+
+            var messageData = (telemetryItems[0].Data.BaseData as MessageData) ?? throw new Exception("Invalid BaseData");
```

Comment:
same as https://github.com/Azure/azure-sdk-for-net/pull/34133/files#r1109171963

Reply:
changed to use Assert.NotNull